sklearn/svm/tests/test_svm.py

```diff
@@ -353,6 +353,7 @@ def test_decision_function_shape():
     assert_equal(dec.shape, (len(X_train), 10))
 
 
+@ignore_warnings
```

Comment:
@amueller : I didn't find another or different test for SVR.predict, although is it a good idea to add/update the tests as a part of this issue?